test/test_services.jl

```diff
@@ -453,3 +453,21 @@ end
         [area_interchange12, area_interchange13, line],
     )
 end
+
+@testset "Test AGC" begin
```

Comment:
@m-bossart we need a test that serializes and de-serializes a system with the AGC and the reserves. I have a sense that it might cause an issue somewhere else